src/pool/rpc.h

```diff
@@ -353,6 +357,15 @@ CRT_RPC_DECLARE(pool_tgt_update_map, DAOS_ISEQ_POOL_TGT_UPDATE_MAP,
 
 CRT_RPC_DECLARE(pool_get_acl, DAOS_ISEQ_POOL_GET_ACL, DAOS_OSEQ_POOL_GET_ACL)
 
+#define DAOS_ISEQ_POOL_PROP_SET	/* input fields */		 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
Ignoring this false positive.